aleth-vm/main.cpp

```diff
@@ -226,6 +226,8 @@ int main(int argc, char** argv)
     if (vm.count("network"))
     {
         string network = vm["network"].as<string>();
+        if (network == "ConstantinopleFix")
```

Comment:
If you're adding it to this tool, then add this possible value to help output, on line 131